libs/examples/README.md

```diff
@@ -0,0 +1,7 @@
+# examples
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
This could use a little more love than the default generation notice 😉

Reply:
some love provided 🥰